### ISSUE TITLE
minimizer removes font definitions

### ISSUE THREAD

**Issue by rondonjon** [webpack/css-loader#94]
I have the following loader defined for all my CSS files:

```
{
   test: /\.css$/,
   loaders: [
       "style",
       "css?minimize"
   ]
},
```

The following CSS is one stand-alone file from font-squirrel.com which defines the DroidSans font:

```
@font-face {
    font-family: 'droid_sansregular';
    src: url('../font/droid_sansregular/DroidSans-webfont.eot');
    src: url('../font/droid_sansregular/DroidSans-webfont.eot?#iefix') format('embedded-opentype'),
         url('../font/droid_sansregular/DroidSans-webfont.woff') format('woff'),
         url('../font/droid_sansregular/DroidSans-webfont.ttf') format('truetype'),
         url('../font/droid_sansregular/DroidSans-webfont.svg#droid_sansregular') format('svg');
    font-weight: normal;
    font-style: normal;
}
```

**Problem**: With the minimizer flag in place, the entire CSS is reduced to an empty string. Removing the minimizer flag resolves the problem.

This applies to all CSS files with @font-faces in them. All other CSS files are imported and minimized as expected.

**Comment by wmertens** [comment]
same here :cry:

**Comment by wmertens** [comment]
This was fixed in 0.15.2, just update your css-loader. I think this issue can be closed.
